server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraDumbBlobStoreCache.java

```diff
@@ -86,24 +84,18 @@ public CassandraDumbBlobStoreCache(Session session, CassandraCacheConfiguration
             insertStatement.bind()
                 .setString(ID, blobId.asString())
                 .setBytes(DATA, data)
-                .setInt(TTL, Long.valueOf(cacheConfiguration.getTtl().getSeconds()).intValue())
+                .setInt(TTL, Math.toIntExact(cacheConfiguration.getTtl().getSeconds()))
```

Comment:
The throwing Math.toIntExact should be done in a Publisher for the save method to return a Mono.error and not throw.

Reply:
I think this cannot be happen because we put validation on this property on `CassandraCacheConfiguration`.

Reply:
Maybe we could add a `// Will never throw as CassandraCacheConfiguration enforces this value to be lower than Integer.MAX_VALUE ` comment to make it clear?

@remk if we can avoid some suboptimal reactor flatMapping wizardry then it can be good...

Reply:
ok, it is good enough for me.
We don't necessarily need to flatmap, we can get the ttl value before hand in a try catch block and return a Mono.error in the throw clause.